src/main/java/io/stargate/sgv2/jsonapi/util/naming/NamingRules.java

```diff
@@ -0,0 +1,10 @@
+package io.stargate.sgv2.jsonapi.util.naming;
+
+public final class NamingRules {
+  private NamingRules() {}
+
+  public static final SchemaObjectNamingRule KEYSPACE = new KeyspaceNamingRule("keyspace");
```

Comment:
So, basically, I think constant "keyspace" can be specified in `KeyspaceNamingRule`, need not be passed from here.